components/dashboard/src/settings/Preferences.tsx

```diff
@@ -190,7 +190,9 @@ export default function Preferences() {
             <div className="mt-4 max-w-md">
                 <h4>Repository URL</h4>
                 <input type="text" value={dotfileRepo} className="w-full" placeholder="e.g. https://github.com/username/dotfiles" onChange={(e) => setDotfileRepo(e.target.value)} />
-                <p className="text-base text-gray-500 dark:text-gray-400">Add a repository URL that includes dotfiles. Gitpod will clone and install your dotfiles for every new workspace.</p>
+                <div className="mt-1">
```

Comment:
Nit: You could add the `mt-1` class to the `<p/>` element below, in order to avoid extra nesting. Not very important though.

Reply:
Tried to replicate the semantics structure we already use elsewhere:

https://github.com/gitpod-io/gitpod/blob/e2c1988ec60a4c73875850ffd7d688bfc230dff8/components/dashboard/src/settings/EnvironmentVariables.tsx#L73-L75

However, I agree we could make this could be improved. Creating a component for this form element could help. 💡